data/json/items/comestibles/sandwich.json

```diff
@@ -301,14 +301,33 @@
     "container": "wrapper",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "calories": 561,
+    "calories": 481,
     "description": "A delicious fish sandwich.",
     "price": 800,
     "material": [ "flesh", "wheat" ],
     "volume": "750 ml",
     "fun": 14,
     "vitamins": [ [ "vitA", 4 ], [ "vitC", 5 ], [ "calcium", 19 ], [ "iron", 17 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "fish_bagel",
+    "name": "fish and spinach bagel",
+    "name_plural": "fish and spinach bagels",
+    "weight": "201 g",
+    "color": "purple",
```

Comment:
this is an invalid color. how about `magenta`

Reply:
Thanks. I change it to magenta.